.github/workflows/ValidatePullRequest.yml

```diff
@@ -5,30 +5,59 @@ name: Validate Pull Request
 on:
   pull_request:
     branches: [main, "release/**"]
-    paths-ignore:
-      - '**.md'
-      - '**.txt'
   merge_group:
 
 permissions:
   id-token: write
   contents: read
 
 jobs:
+  docs-pr:
+    runs-on: ubuntu-latest
+    outputs:
+      not-just-docs: ${{ steps.not-just-docs.outputs.result }}
+    steps:
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            docs:
+              - '**/*.md'
+              - '**/*.txt'
+            all:
+              - '**/*'
+      - uses: actions/github-script@v7
+        id: not-just-docs
```

Comment:
**suggestion:** rename `not-just-docs` to `docs-only` and invert the logic, to avoid the double negation.